backends/ebpf/ebpfstf.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python
+# Copyright 2018 VMware, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+""" Contains functions which allow the parsing of stf files
+    and ultimately generate a C header file which defines the control plane
+    commands for that particular P4 program. """
```

Comment:
Converts the commands in an stf file which populate tables into a C program that manipulates ebpf tables.